src/Ledger/RolloverLedger/useRolloverFiscalYears.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+import {
+  useRolloverFiscalYears,
+} from './useRolloverFiscalYears';
+
+const queryClient = new QueryClient();
+// eslint-disable-next-line react/prop-types
+const wrapper = ({ children }) => (
+  <QueryClientProvider client={queryClient}>
+    {children}
+  </QueryClientProvider>
+);
+
+describe('useRolloverFiscalYears', () => {
+  it('should return undefined if called without series', async () => {
+    const { result, waitFor } = renderHook(() => useRolloverFiscalYears(), { wrapper });
+
+    await waitFor(() => {
+      return !result.current.isLoading;
+    });
+
+    expect(result.current.fiscalYears).toBeUndefined();
+  });
+
+  xit('should return response array', async () => {
+    useOkapiKy.mockClear().mockReturnValue(() => {
+      return ({
+        json: () => {
+          return {
+            data: { fiscalYears: [{ id: 'fyId' }] },
+          };
+        },
+      });
+    });
+
+    const { result, waitFor } = renderHook(() => useRolloverFiscalYears('FY'), { wrapper });
+
+    await waitFor(() => {
+      return Boolean(result.current.fiscalYears);
```

Comment:
@NikitaSedyx could you please look why it can't wait  this happening?

Reply:
should be ok, try to print `result.error` before await, mb there is an error in hook

Reply:
no luck, 
```
    console.log
      result.error undefined
```

Reply:
I give up on this, it works in app, but not in tests. maybe something messed up with mocking or rendering, but result doesn't change.

Reply:
I can checkout and try later

Reply:
thanks a lot!